searcher/src/main/java/ru/otus/searcher/entity/IaMIdiot.java

```diff
@@ -0,0 +1,39 @@
+package ru.otus.searcher.entity;
+
+import dto.SearchResultDto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class IaMIdiot {
```

Comment:
?

Reply:
self-irony... to test "java params by value" - actually it was the one of problem with convert method. 

yes it should be not evn commited to remote branch (no diea why it was).  

deleted,